src/entities/elastic-storage/elastic-storage-processor.js

```diff
@@ -438,6 +438,10 @@ class ElasticStorageProcessor<TxType: ByronTxType | ShelleyTxType> implements St
       refresh: 'true',
       body,
     })
+    if (resp.body.errors) {
+      this.logger.error('bulkUpload error:', resp)
+      throw new Error('bulkUpload failed to upload all documents')
```

Comment:
What happens if we run into this error? Do we have to restart Tangata? Does it auto-correct?

Reply:
it is like `assert`. if it happen, some bug in elastic-tangata is present. There is no way to auto-correct and in most cases restart also will not help.